linera-storage-service/proto/key_value_store.proto

```diff
@@ -139,6 +139,24 @@ message ReplyListAll {
 }
 
 
+message RequestListRootKeys {
+  bytes namespace = 1;
+}
+
+message ReplyListRootKeys {
+  repeated bytes root_keys = 1;
+}
+
+
+message RequestInsertRootKey {
+  bytes namespace = 1;
+  bytes root_key = 2;
+}
+
+message ReplyInsertRootKey {
+}
```

Comment:
(We could use `google.protobuf.Empty` instead of all those empty types. Not sure if that's preferable, but we do it in `rpc.proto`.)

Reply:
Thank you, that is great.